src/core/rendering.js

```diff
@@ -728,7 +728,7 @@ p5.prototype.clearDepth = function(depth) {
  * final color results from blending the source pixel's color with the canvas
  * pixel's color. RGB color values from the source and canvas pixels are
  * compared, added, subtracted, multiplied, and divided to create different
- * effects. Red values with red values, greens with greens, and blues with
+ * effects. #d41159 values with #d41159 values, greens with greens, and blues with
```

Comment:
Shouldn't this stay red, as this statement refers to the color channels, not the color of the line?

Reply:
Good catch!